### PR TITLE
[ko] WebGL fundamentals - WebGL Hello World

### PR DESCRIPTION
**Pull request by TroyTae**
1. Translate WebGL fundamentals - WebGL Hello World
2. Modify language option name from "korean" to "한국어"

### CODE REVIEW

**Comment by TroyTae** [comment]
@greggman 
What is the issue this time?
I added one translated article!

File: webgl/lessons/ko/webgl-fundamentals.md [hunk 1]
```diff
@@ -0,0 +1,610 @@
+제목: WebGL 기초
```
Comment (greggman) [inline webgl/lessons/ko/webgl-fundamentals.md]: The part before the colon's here are keywords. They need to be

```
Title: WebGL 기초
Description: 기초부터 시작하는 WebGL 첫 강의
```

Comment (TroyTae) [reply webgl/lessons/ko/webgl-fundamentals.md]: Oh I see, I will change it.

**Comment by greggman** [comment]
just fyi: you can build locally to test

https://github.com/greggman/webgl-fundamentals/blob/master/README.md#to-build

**Comment by TroyTae** [comment]
@greggman 
Sorry I'll test and upload it from now on.

**Pull request closed without merging**